tests/test_cli.py

```diff
@@ -15,3 +23,102 @@ def test_reporter(capsys, reporter):
     captured = capsys.readouterr()
     assert captured.out == ""
     assert captured.err == "my test\nanother test\n"
+
+
+def test_docs(runner, monkeypatch):
+    calls = []
+    monkeypatch.setattr('webbrowser.open', lambda *args: calls.append(args))
+
+    runner.invoke(cli.docs)
+
+    assert calls == [('https://kitovu.readthedocs.io/en/latest', 2)]
+
+
+class TestEdit:
+
+    @pytest.fixture(autouse=True)
+    def clear_env(self, monkeypatch):
+        monkeypatch.delenv('EDITOR', raising=False)
+
+    class TestMissing:
```

Comment:
Does nesting classes actually work fine with pytest? I've never tried it, but it's something you'd usually not do, so I'd prefer two classes named `TestEditMissing` and `TestEditAvailable` or so.

Reply:
It does work. The advantag is that the clear_env is called in the subclasses as well.

I personally prefer it to be grouped as much as possible, especially with fixtures and before/after hooks it's in my opinion more transparent.